ARM_DisassemblerExplorer/src/parser/ARM/ARM_Instruction.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * @addtogroup  Parser
+ * @{
+ * @file        ARM_Instruction.h
+ * @author      Pascal-Emmanuel Lachance
+ * @p           https://www.github.com/Raesangur
+ * @date        2020/07/24  -  19:51
+ *
+ * @brief       Base pure virtual class onto which all ARM assembly instructions are based.
+ */
+#pragma once
+/*************************************************************************************************/
+#include "parser/ARM/ARM_InstructionSyntax.h"
+#include "parser/Instruction.h"
+
+#include <string_view>
+
+
+namespace Parser
+{
+
+
+/*************************************************************************************************/
+/* Classes ------------------------------------------------------------------------------------- */
+class ARM_Instruction : public Instruction
+{
+public:
+    ARM_Instruction(ARM_InstructionSyntax* syntax)
+    {
+        m_syntax = syntax;
+    }
```

Comment:
can't this be 
```cpp
ARM_Instruction(ARM_InstructionSyntax* syntax)
  : m_syntax(syntax)
{
}
```
?

Reply:
it used to be static, but it's not anymore, and I forgot to put it back in the initialistion list, will do

Reply:
oh nevermind it's still static 😅 
gotta be in the body of the constructor